com.unity.visualeffectgraph/Editor/GraphView/VFXComponentBoard.cs

```diff
@@ -194,10 +194,11 @@ public VFXComponentBoard(VFXView view)
             m_DebugModes = this.Query<Button>("debug-modes");
             m_DebugModes.clickable.clicked += OnDebugModes;
 
+            m_RecordIcon = VFXView.LoadImage("d_Record");
             m_RecordBoundsButton = this.Query<Button>("record");
             m_RecordBoundsImage = this.Query<Image>("record-icon");
```

Comment:
For a small optim I guess it is possible to do `m_RecordBoundsButton.Query<Image>("record-icon");`

Reply:
Done